### ISSUE TITLE
opus.c:  info.max_bps doesnt consider opus_cfg.channel_cnt

### ISSUE THREAD

**Issue by SebaLedesma** [pjsip/pjproject#3265]
### Describe the bug

At https://github.com/pjsip/pjproject/blob/master/pjmedia/src/pjmedia-codec/opus.c the function pjmedia_codec_opus_set_default_param(...) calculates:
param->info.max_bps = opus_cfg.sample_rate * 2;
I think it should be:
param->info.max_bps = opus_cfg.sample_rate * 2 * opus_cfg.channel_cnt;

Also at factory_default_attr(...):
attr->info.max_bps     	   = opus_cfg.sample_rate * 2;
I think it should be:
attr->info.max_bps     	   = opus_cfg.sample_rate * 2 * opus_cfg.channel_cnt;

As stereo configuration requires more bitrate to encode more channels than 1.

### Steps to reproduce

.

### PJSIP version

2.12.1

### Context

.

### Log, call stack, etc

```shell
.
```

**Comment by sauwming** [comment]
True, but I don't think it should be doubled since the audio will be compressed.
In any case, it's just a default value that the user can change, so it should be okay IMO.
